spec/models/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User do
+  # rubocop:disable RSpec/ExampleLength
+  it 'populates a User object' do
+    # rubocop:disable Layout/LineLength
+    dataverse_user_group = 'cn=edu:berkeley:app:calnet-spa:group-spa-ucblibdataverse,ou=campus groups,dc=berkeley,dc=edu'
+    # rubocop:enable Layout/LineLength
+
+    auth = {
```

Comment:
we might be able to shorten this example by calling `let(:auth) { ... }` just outside but within the larger `describe` block

Reply:
Great idea - done!